session_schema.go

```diff
@@ -339,9 +339,15 @@ func (session *Session) Sync2(beans ...interface{}) error {
 					}
 				}
 			}
+
 			if col.Default != oriCol.Default {
-				engine.logger.Warnf("Table %s Column %s db default is %s, struct default is %s",
-					tbName, col.Name, oriCol.Default, col.Default)
+				if (col.SQLType.Name == core.Bool || col.SQLType.Name == core.Boolean) &&
+					((col.Default == "true" && oriCol.Default == "1") ||
```

Comment:
Does casing matter in here? (i.e. is it possible for the value to be `True` or `TRUE`?)

Reply:
done.